classes/Cart.php

```diff
@@ -1521,7 +1522,9 @@ public function updateQty(
         $quantity = (int) $quantity;
         $id_product = (int) $id_product;
         $id_product_attribute = (int) $id_product_attribute;
-        $product = new Product($id_product, false, (int) Configuration::get('PS_LANG_DEFAULT'), $shop->id);
+        $id_customization = (int) $id_customization;
+        $id_address_delivery = (int) $id_address_delivery;
+        $product = new Product($id_product, false, Configuration::get('PS_LANG_DEFAULT'), $shop->id);
```

Comment:
```suggestion
        $product = new Product($id_product, false, (int) Configuration::get('PS_LANG_DEFAULT'), $shop->id);
```

@unlocomqx PHPStan is not happy about having not integer value there :-)